iree/test/e2e/regression/trace_dispatch_tensors.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: iree-run-mlir -iree-hal-target-backends=vmla -iree-flow-trace-dispatch-tensors %s 2>&1 | IreeFileCheck %s
```

Comment:
Not sure how to test on stderr, this is a workaround I have. We can also make it print out the dumps to stdout.

Reply:
I think this pattern is used a bit (with some drawbacks) in LLVM: http://lists.llvm.org/pipermail/llvm-dev/2017-February/110469.html . Seems reasonable enough to me here.

Reply:
Good to know, thanks!